.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,18 @@
+# Apply various labels on PRs
```

Comment:
This labeler was mistakenly removed [here](https://github.com/aws/jsii/pull/3274). The label is needed regardless of the merge mechanism because we use this label in our monitoring automation.